centreon/src/Core/AgentConfiguration/Application/Repository/ReadAgentConfigurationRepositoryInterface.php

```diff
@@ -76,6 +76,17 @@ public function findPollersByType(Type $type): array;
      */
     public function findPollersByAcId(int $agentConfigurationId): array;
 
+    /**
+     * Find pollers by their IDs.
+     *
+     * @param int[] $pollerIds
+     *
+     * @throws \Throwable
+     *
+     * @return Poller[]
+     */
+    public function findPollersByIds(array $pollerIds): array;
```

Comment:
As discussed:
- Move this logic in ReadMonitoringServerRepositoryInterface
- return ?MonitoringServer
- Adapt the SQL Query to only get the central
- Adapt the use case to only check if $central !== null (check isLocalhost and isRemoteServer is no more needed)